src/cluster-map.tsx

```diff
@@ -22,29 +22,29 @@ export class ClusterMap extends React.PureComponent<
   public static defaultProps: Partial<IClusterMapProps> = {
     isClusterExpandClick: true,
   };
-  public mapRef: GoogleMapView;
+  public mapRef: MapView;
   public state: IClusterMapState = {
     markers: [],
     isMapLoaded: false,
     currentZoom: null,
   };
 
   public render() {
-    const { style, region, priorityMarker } = this.props;
+    const { style, region, priorityMarker, provider } = this.props;
 
     return (
-      <GoogleMapView
+      <MapView
         {...utils.serializeProps(this.props)}
         ref={(ref) => (this.mapRef = ref)}
         style={style || styles.map}
         onMapReady={this.onMapReady}
         initialRegion={region}
         onRegionChangeComplete={this.onRegionChangeComplete}
-        provider={PROVIDER_GOOGLE}
+        provider={provider === PROVIDER_DEFAULT ? null : PROVIDER_GOOGLE}
```

Comment:
I think better variant will be 
`provider = { props.provider || PROVIDER_GOOGLE}`
This will cover all user's needs

Reply:
I think the suggested variant won't work for this case, since PROVIDER_DEFAULT value is `null`, `props.provider || PROVIDER_GOOGLE` will always evaluate to PROVIDER_GOOGLE, since `null` evaluates to false

Reply:
You're right. Let's go with your variant